device-types/Datacom/DM4050-24GT-6XS.yaml

```
@@ -0,0 +1,94 @@
+---
+manufacturer: Datacom
+model: DM4050 24GT+6XS
+slug: dm4050-24gt-6xs
+part_number: 8005189xx
+u_height: 1.0
+is_full_depth: true
+airflow: side-to-rear
+comments: L2/L3 wirespeed switch with 24 electrical Gigabit Ethernet ports (RJ45SFP) and 6 optical 10Gisgabit Ethernet ports (SFP+)
+  1U Metal cabinet for 19-inch racks. Two slots for power supplies Power supplies and SFPs to be ordered separately (https://www.datacom.com.br/uploads/product/file/97aede61eba553f23060d983b6e68152.pdf)
```

Seems like this is the change yamlfmt wants

```suggestion
comments: L2/L3 wirespeed switch with 24 electrical Gigabit Ethernet ports (RJ45SFP) and 6 optical 10Gisgabit Ethernet ports (SFP+) 1U Metal cabinet for
  19-inch racks. Two slots for power supplies Power supplies and SFPs to be ordered separately (https://www.datacom.com.br/uploads/product/file/97aede61eba553f23060d983b6e68152.pdf)